src/app/containers/ImageWithPlaceholder/__snapshots__/index.test.jsx.snap

```diff
@@ -42,38 +37,13 @@ exports[`ImageWithPlaceholder should render a lazyloaded image when lazyLoad set
 }
 
-.c1 {
-  display: block;
-  width: 100%;
-  visibility: visible;
-  -webkit-animation: eMLfYp 0.2s linear;
-  animation: eMLfYp 0.2s linear;
-  -webkit-transition: visibility 0.2s linear;
-  transition: visibility 0.2s linear;
-}
-
 <div
-  className="c0"
+  class="c0"
 >
   <div
-    className="lazyload-placeholder"
-    style={
-      Object {
-        "height": undefined,
-      }
-    }
+    class="lazyload-placeholder"
   />
-  <noscript>
-    <img
-      alt="Pauline Clayton"
-      className="c1"
-      height={null}
-      sizes={null}
-      src="https://ichef.bbci.co.uk/news/640/cpsprodpb/E7DB/production/_101655395_paulineclayton.jpg"
-      srcSet="https://ichef.bbci.co.uk/news/640/cpsprodpb/E7DB/production/_101655395_paulineclayton.jpg 640w"
-      width={640}
-    />
-  </noscript>
+  <noscript />
```

Comment:
This tests might require the use of a different renderer, as knowing the `noscript` is set correctly is very important. If someone replaced https://github.com/bbc/simorgh/blob/MoveToLoadableComponents/src/app/containers/ImageWithPlaceholder/index.jsx#L16 with `<noscript />` the tests should fail